src/dialog/dialog.test.tsx

```diff
@@ -4,17 +4,22 @@ import { cleanup, fireEvent, render } from '@config/testing';
 
 import { Dialog } from './dialog';
 
+const id = 'my-dialog';
+const title = 'My dialog';
+const content = 'Message';
+const btnCancelText = 'Cancel';
+const btnConfirmText = 'Confirm';
+let onCancel = jest.fn();
+let onConfirm = jest.fn();
+
 describe('Dialouge', () => {
```

Comment:
Could you please fix this typo while you are on this ? It should be `Dialog` instead of `Dialouge`.